governance.md

```diff
@@ -0,0 +1,105 @@
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+## Maintainer
+
+Each project has one or more *Maintainers*. Project maintainers are first and
+foremost active *Contributors* to the project and are responsible for:
+
+* Setting technical directions for the project.
+* Facilitating, reviewing and merging contributions. They have write access to
+  the project repository.
+* Creating and assigning project issues.
+* Enforcing the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+The list of maintainers for a project is defined by the project
+`CODEOWNERS` file.
+
+### Becoming a project maintainer
+
+Existing maintainers may decide to elevate a *Contributor* to the *Maintainer*
+role based on the contributor established trust and contributions relevance.
+This decision process is not formally defined and is based on lazy consensus
+from the existing maintainers.
+
+Any contributor may request for becoming a project maintainer by opening an
+pull request (PR) against the `CODEOWNERS` file, and adding *all* current maintainers
+as reviewer of the PR.
+Maintainers may also pro-actively promote contributors based on their
+contributions and leadership track record.
+
+## Steering Committee Member
+
+The Steering Committee (SC) is the overall Confidential Containers organization
+governing body.
+
+The SC provides decision-making and strategic oversight for the project. It also
+defines and enforces the project values and structure.
+
+### Scope and Responsibilities
+
+The scope and responsibilites of the SC is subject to changes, and SC members must
+adapt it to meet the project needs.
+Moreover, any technical responsibilities should be delegated to project Maintainers,
+although the SC can be consulted to help the with making technical decisions.
+
+Based on that, the SC is responsible for:
+
+* Defining the project high-level strategy and roadmap
+* Managing and administrating the project
+* Building and growing the Confidential Containers community
+* Onboarding and guiding Confidential Containers end-users
+
+### Members
+
+The initial composition of the SC is self-defined and is made of 10 members:
+
+* Larry Dewey (@larrydewey) - AMD
+* Jiang Liu (@jiangliu) and Jia Zhang (@jiazhang0) - Alibaba
+* James Magowan (@magowan)  and Tobin Feldman-Fitzthum (@fitzthum) - IBM
+* Peter Zhu (@peterzcst) and Dan Middleton (@dcmiddle) - Intel
+* Pradipta Banerjee (@bpradipt)  and Ariel Adam (@ariel-adam) - Red Hat
+* Samuel Ortiz (@sameo)
+
+#### Selection
+
+The initial SC members will be responsible for defining how and when the SC
+membership can be expanded or renewed.
+
+### Quorum
+
+Each SC member is expected to attend each SC meeting, and the following guidelines
+are used to determine if quorum is reached:
+
+* Quorum to meet is 1/2 (5/10)
+* Quorum to vote is 2/3 (7/10)
```

Comment:
Do we need to define what we are voting on? This is defining how a vote should be accepted, if the SC decides to vote on something. IMHO, voting should be a last resort kind of thing. We could define a process for any member to call for a vote, but that's pretty heavy at that point.

And yes, SC members should start right away working on amending that document with anything they believe should be added (Although I'm hoping discussions on that PR should lead us to something all 10 members largely agree on), and in particular define the SC renewal/extension process.

Reply:
Should we have an odd number of members?

Reply:
> Should we have an odd number of members?

This brings ups an important point. If we have a split decision, having an even number of committee members could potentially stall during the vote.

Reply:
Note that the quorum for voting isn't the full committee. Regardless of the size of the committee there is no guarantee that an odd number of people would show up to vote.

Reply:
> Regardless of the size of the committee there is no guarantee that an odd number of people would show up to vote.

That is a valid point. It is likely that most scenarios at least one member will likely be missing. That being said, I think it is probably a good idea to have prepared for it before hand.

> I guess it's not ideal if everyone were to show up and the committee ends up split, but this doesn't keep me up at night.
 
It doesn't keep me up, either, it was more of an observation. I just wouldn't want a split committee preventing the progress of the project. Given the character of the members, and the fact that it hasn't really been an issue up to this point, I would venture to guess it won't be a major issue in the near future.

Reply:
I agree with @fitzthum here, basically the SC size is vaguely related to the number of votes.

Moreover, and given the scope of the SC and at least its initial composition, if we end up in a situation where it's split on a given decision, it fundamentally means there is a community disagreement. Personally, I am not in favor of resolving such contention through e.g. a 6 vs 5 voting result (Assuming a 11 SC members). The fundamental role of the SC in that case would be more about finding a way out of it that does not potentially leave almost half of the community unhappy with a decision.

And yes, I dont expect this is a situation that we'll face often, if ever. So, in the spirit of keeping things simple, I'd prefer to not be too formal with those details.

Reply:
I like the description of consensus driven decision making in the Rust community. 
https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md#consensus
It would be good to add similar language here. The style of our document is much more succinct, but perhaps we can link to broader text like Rust's.

Reply:
Thanks for the suggestion @dcmiddle. I added a small section about decision-making.

Reply:
I like the wording at present as a good starting point and I think any "vote" that is close should lead to further discussion - and if the steering group itself cannot resolve it then I wouldn't see an issue encouraging and inviting  all in community to discuss and help reach a decision? This will hopefully never be needed but can be our last resort option?
Ultimately there is no last resort great option here , we can't control the numbers attending any meeting and if a vote is close unless we address the major concerns any decision will be less sticky and no doubt come back again in near future.